desci-models/src/ResearchObject.ts

```diff
@@ -50,6 +50,7 @@ export interface ResearchObjectV1Component {
   id: string;
   name: string;
   type: ResearchObjectComponentType | ResearchObjectComponentTypeMap;
+  subtype?: string;
```

Comment:
this overrides
<img width="795" alt="Screenshot 2024-01-18 at 4 41 08 AM" src="https://github.com/desci-labs/nodes/assets/278886/e2515b51-17a4-4fa7-8e77-194df95b4f89">

Reply:
I would suggest naming this string field customSubtype.